RELEASE_NOTES.md

```diff
@@ -0,0 +1,159 @@
+# BEAM Release Notes
+## 1.0.0
+### New Features
+#### Ride-hail features
+- DemandFollowingRepositionig: Reposition to the random activity's location.
+- Inverse square demand following repositioning.
+- Wheelchair accessible ride-hail vehicles.
+- Ride-Hail CAVs charging.
+- Introduced ride-hail stops.
+- Ride-hail name in OD skims.
+- Configuring allowed ride-hail modes.
+- RideHail skimmer.
+
+#### Mode choice feature
+- Allowing to do a mode choice if the current mode cannot be implemented.
+- Improved tour mode choice.
+
+#### Router features
+- R5: Limit routing by max distance for BIKE.
+- Introduced GraphHopper router.
+- Not found routes metric.
+- Introduced Native CCH router.
+- Send routing failures to skims.
+
+#### Parking features
+- Hierarchical parking manager to improve parking stall search performance.
+- Link based parking sampling.
+- Parking Manager on the link level.
+- Improved performance of parking and charging manager.
+- Assign parking stalls to households.
+- Parking skimmer.
+- Include parking in activity generation.
+- New parking stall search algorithm.
+
+#### Freight features
+- Freight simulation.
+- Freight origin-destination skims.
+- Vehicle energy rate based on the weight of the truck + payload.
+
+#### Other features
```

Comment:
I categorized all other features as follows - please review/changes as needed:

### Physsim

- Possibility to write PhysSim events to CSV.
- PhysSim XML to OSM converter.
- New PhysSim mode: BPR.
- Performance: Parallel Event Manager in PhysSim.
- Added option to convert PhysSim network to PBF directly.
- Artificial increasing of traffic flow for better PhysSim.
- Allow choosing an event manager for PhysSim.



### Scripts / Postprocessing:
- Introduced Jupyter notebooks in Beam source code.
- Jupyter notebook for map analysis of osm.pbf and PhysSim network.
- Jupyter notebook for linkstats and network.csv files analysis.
- Jupyter notebook to adjust GTFS time and timezone and to downsample scenarios based on a shape file.
- Output analysis with python scripts.
- A script for filtering points from shapefile.
- Beam Health Analysis script.
- Frequency converter utility
- Introduced a possibility to execute a Jupyter server along with Beam.
- Added various graphs/statistics.
- A tool to get a shape file out of stops coordinates from GTFS archive.
- Matsim network to shape file converter.
- Travel time & distance calculator app.
- Util to compare values of two config files.
- Utility to create TAZ geofences out of shapefiles.
- Possibility to read a shape file as TAZ file.

### Modes
- Mode BIKE_TRANSIT is added.
- Special treatment of bike lanes.
- Simulating preference to the transit routes with lower crowdedness.
- Shared vehicles on route egress.
- HOV2 and HOV3 modes from activitysim plans.
- Shared bike repositioning.
- Include intercepts in mode utilities.
- Train station car usage.
- Micromobility modeling.
- Add buffer time for access to transit.
- Allow emergency vehicles for HOV modes.
- Create household cars from Vehicle types.
- Added trip id to transitVehicleTypesByRoute File.

### Skims
- Generate skims for provided origin-destination pairs and modes.
- Producing activitysim skims at runtime.
- Keep track of wait times in skims.
- More skimmer events.
- Use TAZ geometries for skim origin/destination.
- Throw skimmer events for non-chosen itineraries.
- Use non-generalized times and costs in asim skimmer.
- Activitysim skims in omx format.


### Electric Vehicles
- Charging compatibility is taken into consideration.
- Co-simulation of power providing/consuming with HELICS lib.
- In-simulation charging scale up.
- Initial State of Charge for beam vehicles.
- En-route charging.

### Refactoring
- Removes deprecated parameters from configurations.
- Documentation improvements.
- Input Consistency Check.


### Activities / Plans
- Generate secondary activities.
- Remove activities/persons for certain industries.
- Allow to merge the urbansim input plans with the last BEAM run output plans.
- Fixed activity duration based on the activity name.
- Create separate plans based on planIndex and set "selected" only (valid) one of them.
- Create and store a network route for a personal plan if it is empty.
- Force vehicle creation when plans require it.

### Debugging / Logging
- Improved simulation got stuck detection.
- Storing the scheduler state in case of Beam exists with an error.
- Beam internal message logging.
- Health metrics analysis in beam output directory.

### UrbanSim
- Urbansim mode mapping from config.
- Include urbansim trip_id as an attribute in beam plans.
- Get leg mode directly from Urbansim plans.
- Configurable timeout for urbansim scenario loading.

### Deployment
- Add a possibility to automatically run Beam on Lawrencium HPC.
- Add a possibility to automatically run Beam on Google Compute Engine.
- Deployment spreadsheet improvements.